### ISSUE TITLE
Issue Displaying city selector ACF Field Type in Elementor

### ISSUE THREAD

**Issue by vincent11235** [Beee4life/acf-city-selector#137]
Hi [Beee4life](https://github.com/Beee4life),

I am using the ACF Form widget in Elementor, provided by the Frontend Admin plugin by Dynamiapps. While I can successfully display built-in field types on the frontend form, I am encountering an issue with the field type "city selector" you built. The dropdown menu for this field does not appear on the frontend, unlike in the backend where it displays correctly.

Attached are the screenshots for reference:

Frontend View (missing dropdown): <img width="1130" alt="image" src="https://github.com/user-attachments/assets/90e4b62c-0107-490f-b3de-6e92954bf3a1">
Backend View (dropdown visible): <img width="1359" alt="image" src="https://github.com/user-attachments/assets/dcf66416-4fb9-467c-be5e-776b2a7989dc">
HTML in dev tools: 
<img width="398" alt="image" src="https://github.com/user-attachments/assets/4f09952e-c34d-4351-a463-cbbbee546ba8">

Could this issue be related to how I've configured the field type in ACF, or might it be a problem with CSS and JavaScript not properly applying on the front end? Any guidance on troubleshooting or resolving this issue would be greatly appreciated.

Thank you for your assistance.

**Comment by Beee4life** [comment]
Hi @vincent11235 

Let start by saying, I hate elementor.

I noticed it can give issues as reported by another user, but I am not planning on doing anything about it. Those tools (Elementor, Salient, WP Bakery, etc.) are a nuisance for some scripts.

The form does work on front-end situations. Check the website for an example.

Elementor is messing it up, I am sorry.

**Comment by vincent11235** [comment]
I've determined that the issue was not with Elementor but with Frontend Admin. 

The class "frontend-admin-hidden" was supposed to apply only to the sections I wanted hidden. Ultimately, I resolved the issue by forcing the CSS for the class "frontend-admin-hidden" from display: none to display: block in order to make the city selector visible on the frontend. 

This solution isn't ideal, but it effectively addresses the problem for now.
